### ISSUE TITLE
ForbiddenMethodCall not working for @Composable getter

### ISSUE THREAD

**Issue by WildOrangutan** [detekt/detekt#7877]
## Expected Behavior

I'm trying to blacklist [MaterialTheme.shapes](https://cs.android.com/androidx/platform/frameworks/support/+/androidx-main:compose/material3/material3/src/commonMain/kotlin/androidx/compose/material3/MaterialTheme.kt;l=136?q=MaterialTheme%20compose) property, but it doesn't seem to be working.

Afaik property can be blacklisted by using `ForbiddenMethodCall`, please correct me if I'm wrong.

I've tried using following:

```yml
style:
  ForbiddenMethodCall:
    active: true
    methods:
      - value: androidx.compose.material3.MaterialTheme.getShapes
        reason: "Use AppShapes instead"
      - value: androidx.compose.material3.MaterialTheme.shapes
        reason: "Use AppShapes instead"
```

I'm using `detektMain` task to run lint with type resolution.

## Observed Behavior

Lint does not fail, when using rules above

## Steps to Reproduce

Add rule above for @Composable getter and run linter.

## Your Environment
* Version of detekt used: 1.23.7
* Version of Gradle used (if applicable): 8.9
* Operating System and version: Ubuntu latest
